zio/zsonio/ztests/first-test.yaml

```diff
@@ -0,0 +1,118 @@
+zql: "*"
```

Comment:
Can we get a better more descriptive test than `first-test.yaml`? Ideally shouldn't these be broken up by test case?

Reply:
first-test.yaml is an instant classic!